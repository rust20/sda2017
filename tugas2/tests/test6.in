5 2 2 50 5 5
Enru Saver;20;120;pedang
Queen Riana;30;120;pedang
Kuroneko;10;1000;panah
Ikhlas Al-Ampas;10;999;panah
p1scok;10;1001;panah
1;4;Enru Saver,Queen Riana
3;4;Kuroneko,Ikhlas Al-Ampas,p1scok
2;5;1;80;pedang;1
4;1;2400;160;pedang;2
M..S.
####D
...S.
D####
.....

