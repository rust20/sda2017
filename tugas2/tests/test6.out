1,4 Pahlawan yang ikut:Enru Saver,Queen Riana
2,5 BATTLE, kekuatan: 120, pahlawan: Enru Saver
3,4 Pahlawan yang ikut:p1scok,Kuroneko,Ikhlas Al-Ampas
4,1 BATTLE, kekuatan: 4002, pahlawan: p1scok,Kuroneko
Akhir petualangan Gudako
Level Gudako: 401
Level pahlawan:
p1scok: 161
Kuroneko: 161
Enru Saver: 81
Queen Riana: 1
Ikhlas Al-Ampas: 1
